homeassistant/components/logbook/websocket_api.py

```diff
@@ -395,9 +386,22 @@ def _queue_or_cancel(event: Event) -> None:
         partial=True,
     )
 
-    await _async_wait_for_recorder_sync(hass)
+    live_stream.task = asyncio.create_task(
```

Comment:
You can put this below `if msg_id not in connection.subscriptions:`. There is no awaits happening

Reply:
Yup, and I can drop the unsub call as well since we know it already happened

Reply:
a63d6eb229540090c216b5c2ab397a28f5bd9c19